src/shogun/machine/Machine.h

```diff
@@ -147,7 +142,7 @@ enum EProblemType
  * locking.
  *
  */
-class CMachine : public CSGObject
+class CMachine : public CStoppableSGObject
```

Comment:
@vigsterkr @lisitsyn can we start using multiple inheritance in the c++ framework? Or is there still a blocker?